__tests__/app.test.js

```diff
@@ -69,4 +70,64 @@ describe("GET /api/articles/:article_id", () => (
             })
         })
     })
-));
\ No newline at end of file
+));
+
+describe("PATCH /api/articles/:article_id", () => {
+    describe("Connecting to path", () => {
+        test("Should return status 200 ", () => {
+            return request(app).patch("/api/articles/1").send({ inc_votes : 1 }).expect(200)
+        })
+        test("Should return the updated article if patch successful: increase votes", () => {
+            const testUpdate = {
+                title: "Living in the shadow of a great man",
+                topic: "mitch",
+                author: "butter_bridge",
+                body: "I find this existence challenging",
+                created_at: "2020-07-09T20:11:00.000Z",
+                votes: 101,
+                article_img_url:
+                  "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+            }
+            return request(app).patch("/api/articles/1").send({ inc_votes: 1 })
+                .then(({ body: { updatedArticle } }) => {
+                expect(updatedArticle).toMatchObject(testUpdate)
+            })
+        })
+        test("Should return the updated article if patch successful: decrease votes", () => {
+            const testUpdate = {
+                title: "Living in the shadow of a great man",
+                topic: "mitch",
+                author: "butter_bridge",
+                body: "I find this existence challenging",
+                created_at: "2020-07-09T20:11:00.000Z",
+                votes: 99,
+                article_img_url:
+                  "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+            }
+            return request(app).patch("/api/articles/1").send({ inc_votes: -1 })
+                .then(({ body: { updatedArticle } }) => {
+                expect(updatedArticle).toMatchObject(testUpdate)
+            })
+        })
+    })
+    describe("PATCH Errors", () => {
+        test("Should return status 400 for invalid request ", () => {
+            return request(app).patch("/api/articles/invalid").send({ inc_votes: 1 }).expect(400)
+                .then(({ body: {msg} }) => {
+                expect(msg).toBe("Bad Request")
+            })
+        })
+        test("Should return status 400 for valid request with incorrect property", () => {
+            return request(app).patch("/api/articles/invalid").send({ invalid: 1 }).expect(400)
+                .then(({ body: {msg} }) => {
+                expect(msg).toBe("Bad Request")
```

Comment:
Interestingly I think you could design you API to be more open to expansion by ignoring unnecessary properties. If one day we want to allow users to PATCH the title or the body, we would want this test to be open to that and not break. 
It's possible that if the inc_votes is missing we could just expect a `200` response and have the article be _unchanged_ 🤔